.github/workflows/build_vagrant.yml

```diff
@@ -25,7 +25,7 @@ jobs:
     - uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
 
     - name: Update Repos
-      run: sudo apt-get update
+      run: sudo apt-get update || true
```

Comment:
Why this change?

Reply:
Temporary fix for the broken ubuntu runner action... I forgot to remove it... I'll do it now